src/components/Character.tsx

```diff
@@ -0,0 +1,40 @@
+import {View, Text, StyleSheet} from 'react-native';
+import React from 'react';
+
+type characterProps = {
+    character: {
+        hasOwned: boolean,
+        isUsed: boolean,
+        imageUrl: string,
+    }
+}
```

Comment:
[nit] Suggest extracting character type:

```suggestion
type Character = {
  hasOwned: boolean;
  isUsed: boolean;
  imageUrl: string;
};

type CharacterProps = {
  character: Character;
};
```

Reply:
1065ebc